modules/addons/src/gltf/gltf-environment.js

```diff
@@ -1,11 +1,13 @@
 import GL from '@luma.gl/constants';
-import {Texture2D, TextureCube, loadImage} from '@luma.gl/core';
+import {Texture2D, TextureCube} from '@luma.gl/core';
+import {loadImage} from '@loaders.gl/images';
 
 export default class GLTFEnvironment {
-  constructor(gl, {brdfLutUrl, getTexUrl}) {
+  constructor(gl, {brdfLutUrl, getTexUrl, specularMipLevels = 10}) {
```

Comment:
Possible to add any docs or just a single line of comment? To be fair I don't think this class was ever well documented, but still good to share the thinking behind this new parameter for maintainers.

Reply:
Will do. FWIW, the API for this class strikes me as very janky. The string template to pass URLs means the code can't know ahead of time how many images are coming in. If we change that to passing an array or URLs, that problem would go away.